robottelo/ui/hosts.py

```diff
@@ -8,138 +8,193 @@
 class Hosts(Base):
     """Provides the CRUD functionality for Host."""
 
-    def _configure_hosts(self, domain, subnet, puppet_ca=None,
-                         puppet_master=None, host_group=None,
-                         resource=None, env=None, ip_addr=None,
-                         mac=None, os=None, arch=None, media=None,
-                         ptable=None, custom_ptable=None, root_pwd=None):
+    def _configure_hosts(self, arch=None, custom_ptable=None, cv=None,
+                         domain=None, env=None, host_group=None, ip_addr=None,
+                         lifecycle_env=None, mac=None, media=None, os=None,
+                         ptable=None, puppet_ca=None, puppet_master=None,
+                         resource=None, root_pwd=None, subnet=None):
+        # Host tab
+        if lifecycle_env:
```

Comment:
Better to check explicitly for `None`, because eventually I may want to pass a empty value like `''` if available on the select values.